pwa-devdocs/src/venia-ui/reference/components/Link/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Link
+---
+
+# Link
+
+The `Link` component wraps the [react-router-dom Link](https://knowbody.github.io/react-router-docs/api/Link.html) component, but adds intelligent prefetching behavior to speed up performance. Our `Link` component binds to an `IntersectionObserver` to prefetch the target page only when the link is visible in the viewport. And when possible, our `Link` loads the page from local cache to prevent unnecessary roundtrips to the server. In short, the `Link` component builds in all the best-practices that Google recommends when prefetching content for fast transitions.
```

Comment:
> to prefetch the target page only when the link is visible in the viewport. And when possible, our `Link` loads the page from local cache

Should be changed to
> to prefetch the target page's type information only when the link is visible in the viewport. And when possible, our `Link` loads the page information from local cache

Or something similar. Just so we clarify that we're not fetching _all_ of the page's content, but just information about its type (ex that it's a product page) so the transition is quicker